### PR TITLE
chore(docs): fix rule wording and grammar

### PR DESCRIPTION
**Pull request by quixoticmonk**
Updates some of the info and messages across AWS solutions /Hipaa rules.

### Question :
- AwsSolutions-OS5:
  - Should this be `The OpenSearch Service domain allows for unsigned requests or anonymous access.` than ` The OpenSearch Service domain does not allow for unsigned requests or anonymous access.`

### CODE REVIEW

**Comment by dontirun** [comment]
> * Should this be `The OpenSearch Service domain allows for unsigned requests or anonymous access.` than ` The OpenSearch Service domain does not allow for unsigned requests or anonymous access.`

It should be the former `The OpenSearch Service domain allows for unsigned requests or anonymous access.`

**Comment by quixoticmonk** [comment]
The `OS5` changes are added to reflect the updated info message.

File: src/packs/hipaa-security.ts [hunk 1]
```diff
@@ -310,9 +310,9 @@ export class HIPAASecurityChecks extends NagPack {
       node: node,
     });
     this.applyRule({
-      info: 'The CodeBuild project which utilizes either a GitHub or BitBucket source repository does not utilize OAUTH - (Control IDs: 164.308(a)(3)(i), 164.308(a)(4)(ii)(A), 164.308(a)(4)(ii)(C), 164.312(a)(1)).',
```
Comment (dontirun) [inline src/packs/hipaa-security.ts:313]: For consistency can we also update the description for these two packs (and their RULES.md entries)?

1. https://github.com/quixoticmonk/cdk-nag/blob/rules_info_wording/src/packs/nist-800-53-r4.ts#L263
2. https://github.com/quixoticmonk/cdk-nag/blob/rules_info_wording/src/packs/pci-dss-321.ts#L279

File: src/packs/hipaa-security.ts [hunk 2]
```diff
@@ -750,7 +750,7 @@ export class HIPAASecurityChecks extends NagPack {
       node: node,
     });
     this.applyRule({
-      info: 'The RDS DB instance does not enhanced monitoring enabled - (Control ID: 164.312(b)).',
```
Comment (dontirun) [inline src/packs/hipaa-security.ts:753]: Same request for this one

1. https://github.com/quixoticmonk/cdk-nag/blob/rules_info_wording/src/packs/nist-800-53-r4.ts#L626
2. https://github.com/quixoticmonk/cdk-nag/blob/rules_info_wording/src/packs/nist-800-53-r5.ts#L696

**Review by dontirun — CHANGES_REQUESTED** [review]

**Review by dontirun — APPROVED** [review]

File: src/packs/hipaa-security.ts (hunk 2 above)
Comment (quixoticmonk) [reply src/packs/hipaa-security.ts:753]: All of these are updated as requested.

**Comment by dontirun** [comment]
Thanks for the contribution! @quixoticmonk

**Pull request merged**